src/main/webapp/index.jsp

```diff
@@ -37,7 +37,7 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+      <h2>Welcome to our chat!</h2>
```

Comment:
This change seems irrelevant to the rest of the PR. To build good habits, remove if unintended change. Otherwise, change in new PR.

Not a big deal here but generally speaking, we want to keep PRs scoped. For example, if profile page changes introduce an error, we could want to be able to revert this PR without reverting other unrelated changes.